pkg/model/clusterconfigentity.go

```diff
@@ -120,6 +120,7 @@ type ReconciliationSequence struct {
 type ReconciliationSequenceConfig struct {
 	PreComponents  [][]string
 	DeleteStrategy string
+	DesiredStatus  Status
```

Comment:
I would suggest a different name for this field, it is not a status that needs to be reached but actually the status the reconciliation for which we are creating the sequence is in right now.

Maybe something like `ReconciliationStatus` would be better.